packages/common/src/chains/mainnet.json

```diff
@@ -74,6 +74,11 @@
       "name": "berlin",
       "block": 12244000,
       "forkHash": "0xeb440f6"
```

Comment:
Just a side note that this berlin forkhash doesn't seem to be correct with this missing 0 at the beginning of the byte sequence. I ran into some mainnet connection problems in some other context I can't remember with the other client expecting a `0x0eb440f6`. Just mentioning here since this might have caused some other bugs and/or needed fixes on this specific PR.

Reply:
Additional note (we might want to open a dedicated issue to discuss): apart from an eventual direct fix here this might also need a correction of the `forkHash` creation logic in `Common`.

Reply:
Huh this is weird, I thought I fixed this, also had a similar problem in Yolo I think.